packages/redux/src/orders/selectors.ts

```diff
@@ -410,7 +412,7 @@ export const getOrderReturnOptionsError = (
  * @returns Tracking Loading status.
  */
 export const isTrackingsLoading = (state: StoreState) =>
```

Comment:
Same as the other comment for renaming this to `areShipmentTrackingsLoading`.

Reply:
✅